tests/test_po.py

```diff
@@ -56,11 +56,8 @@ class TestGTranslator (unittest.TestCase):
     def test_gtranslator (self):
         """Test all pofiles for GTranslator brokenness."""
         for f in get_pofiles():
-            fd = file(f)
-            try:
+            with open(f, 'rb') as fd:
                 self.check_file(fd, f)
```

Comment:
On Python 2 this changes newline interpretation, which doesn't matter for `check_file`.

On Python 3 this changes a NameError for `file` into a TypeError: startswith first arg must be bytes or a tuple of bytes, since `check_file()` assumes it'll get text, not bytes.

I suggest a couple of `b` string prefixes in check_file, and it's all good.